app/Controller/SongsController.php

```diff
@@ -194,8 +194,8 @@ public function albums() {
                 'Song' => array(
                     'fields' => array('Song.id', 'Song.band', 'Song.album', 'Song.cover'),
                     'conditions' => $subQuery,
-                    'order' => 'Song.album',
-                    'limit' => 36
+                    'order' => ['Song.band', 'Song.album'],
+                    'limit' => 60
```

Comment:
That's a LOT of hacking. I dived into the code and found a solution a bit more elegant and without breaking anything else: https://github.com/ltGuillaume/sonerezh/commit/bfb12291dc0b669d6bf300abd81a64bdb6780918